src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from 'react-router-dom';
+import { StyledNavBar } from '../styles/NavBar.styled';
+import navbarImage from '../../assets/navbar/navbar_photo.png';
+import SearchBar from '../SearchBar/SearchBar';
+
+export default function NavBar() {
+  const onSearch = () => {};
+  return (
+    <>
+      <StyledNavBar>
+        <div className="NavBar__logo">
+          <Link className="link" to="/">
+            WizeTube
+          </Link>
+          <SearchBar onSubmit={onSearch} />
+        </div>
+        <div className="NavBar__right">
+          <button className="Navbar__button">Dark Mode</button>
+          <img className="NavBar__right__img" src={navbarImage} alt="" />
+        </div>
+      </StyledNavBar>
+    </>
```

Comment:
- The empty component is not needed
- Instead of creating css classes what if you change the approach and for example, you create a new styled component named <Logo> in which you can encapsulate the logic of the WizeTube logo? (Same for the Image), in this approach you are going to have small components but at some point you can reuse it

```suggestion
      <StyledNavBar>
        <div className="NavBar__logo">
          <Logo/>
          <SearchBar onSubmit={onSearch} />
        </div>
        <div className="NavBar__right">
          <button className="Navbar__button">Dark Mode</button>
          <Image source={navbarImage} />
        </div>
      </StyledNavBar>
```